vignettes/pkgnet-intro.Rmd

```diff
@@ -0,0 +1,354 @@
+</div>
+
+<!--  SIDE BY SIDE END -->
+
+<br><br>
+
+#### Check out the full HTML report for more results  
+
+## The List Object
+
+The `CreatePackageReport()` function returns a list with three items: 
+
+1. PackageSummaryReporter  
+2. PackageDependencyReporter  
+3. PackageFunctionReporter  
+
+Each items contains information visible in the report *and more*.  We can use this information for a more detailed analysis of the results and/or more easily incorporate `pkgnet` results into other R processes.
+
+Here are a few notable items available within the list object.
+
+### Node Information
+Both the `PackageDependencyReporter` and the `PackageFunctionReporter` contain metrics about their package dependencies or functions (a.k.a network nodes) in a `nodes` table.  
+
+```{r nodes}
+dim(report2$PackageFunctionReporter$nodes)
+names(report2$PackageFunctionReporter$nodes)
+```
+
+### Network Measures 
+Both the `PackageDependencyReporter` and the `PackageFunctionReporter` contain measure based on their network structure in a `network_measures` list. 
+
+```{r networkMeasures}
+report2$PackageFunctionReporter$network_measures
+```
+
+### Network Object
+Both the `PackageDependencyReporter` and the `PackageFunctionReporter` are available as [igraph](http://igraph.org/r/) objects named `pkg_graph`
+
+```{r networkObj}
+report2$PackageFunctionReporter$pkg_graph
+```
+
+***
+# A Deeper Look
+With the reports and objects produced by `pkgnet` by default, there is plenty to inform us on the inner workings of an R package.  However, we may want to know MORE! Since the [igraph](http://igraph.org/r/) objects are available, we can leverage those graphs for further analysis.  
+
+In this section, let's examine a larger R package, such as [lubridate](http://lubridate.tidyverse.org/).
+
+If you would like to follow along with the examples in this section^[Examples from version 1.7.3 of Lubridate], run these commands in your terminal to download and install `lubridate`.  
+```{r bashText, engine='bash', eval=FALSE}
+# Create a temporary workspace
+mkdir -p ~/pkgnet_example && cd ~/pkgnet_example
+
+# Grab the lubridate source code
+git clone https://github.com/tidyverse/lubridate
+
+# Install it
+cd lubridate
+Rscript -e "devtools::install()"
+```
+
+## Coverage of Most Referenced Functions
+Let's visualize the relationship between function centrality and code coverage.
+
+<!--  Faked Since lubridate not guaranteed to be installed at time of report creation -->
+```{r fakeDetail1, eval=FALSE}
+# Run pkgnet
+library(pkgnet)
+report2 <- CreatePackageReport(pkg_name = "lubridate"
+                                         , pkg_path = "~/pkgnet_example/lubridate")
+
+# Extract Nodes Table
+funcNodes <- report2$PackageFunctionReporter$nodes
+
+# List Coverage For Most Referenced Functions
+mostRef <- funcNodes[order(numDescendants, decreasing = TRUE)][1:10]
+mostRef[,list(node, numDescendants, coverageRatio, totalLines)]
```

Comment:
Any reason we don't use the standard `data.table` `.(...)` here instead of `list(...)`?